src/convertToObject.js

```diff
@@ -6,7 +6,22 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const array = sourceString.split(';');
+
+  const object = array.reduce((accumulator, current) => {
+    const element = current.split(':');
+
+    if (element.length === 2) {
```

Comment:
[CODE PRACTICE] - This check `if (element.length === 2)` is fine, but it might be better to filter out invalid entries before the `reduce` function. This way, the `reduce` function will only deal with valid entries, making the code cleaner and easier to read.